src/Setup/DevDivVsix/MicrosoftCodeAnalysisLanguageServices/MicrosoftCodeAnalysisExpressionEvaluator/Microsoft.CodeAnalysis.ExpressionEvaluator.swr

```diff
@@ -16,6 +16,8 @@ vs.payloads
 
 vs.dependencies
   vs.dependency id=Microsoft.VisualStudio.MinShell
+  vs.dependency id=Microsoft.Net.4.6.1.FullRedist 
```

Comment:
Small Nit: Ordering can matter in these dependencies, so if you want to make sure .NET installs before MinShell, this should go before MinShell like it does in the other packages.